0.10/Dockerfile.rhel7

```diff
@@ -32,6 +35,8 @@ COPY ./s2i/bin/ $STI_SCRIPTS_PATH
 # run and build the applications.
 COPY ./contrib/ /opt/app-root
 
+RUN $PROMPT_FILE && npm install -g nodemon
```

Comment:
@humaton, can you take a look at this PR, pls?

As for the inclusion of `nodemon` in SCL - we should track this requirement for at least next node collection -- @bparees, can I ask you for filling up a report with stating the rationale? https://bugzilla.redhat.com/enter_bug.cgi?product=Red Hat Software Collections&component=distribution&version=nodejs010

Since adding new packages is not done via z-streams, and there will most probably be newer version of node next release, I think the best way to go now is to include own build for existing nodejs collection in OSE repository, same as it was done for python33-pip.